crates/types/src/rpc/transactions/broadcasted_declare_transaction_v2.rs

```diff
@@ -128,22 +129,23 @@ mod tests {
     }
 
     #[test]
-    fn sierra_hash_from_events_sierra_artifact() {
+    fn compare_sierra_hash_from_events_sierra_artifact_with_starknet_in_rust_implementation() {
         let sierra_contract_path =
             concat!(env!("CARGO_MANIFEST_DIR"), "/test_data/events_cairo1.sierra");
 
         let cairo_1_contract = ContractClass::cairo_1_from_sierra_json_str(
             &std::fs::read_to_string(sierra_contract_path).unwrap(),
         )
         .unwrap();
-        let sierra_class: SierraClass =
-            serde_json::from_value(serde_json::to_value(cairo_1_contract.clone()).unwrap())
-                .unwrap();
-        println!("{}", Felt::from(sierra_class.class_hash().unwrap()).to_prefixed_hex_str());
-
-        println!(
-            "{}",
-            Felt::from(compute_sierra_class_hash(&cairo_1_contract).unwrap()).to_prefixed_hex_str()
+
+        assert_eq!(
```

Comment:
What happened in this test? Why are we only adding some assertions now? Also, are we keeping starknet_in_rust as a dependency for these couple of tests? Should the dependency then be labeled in Cargo.toml as used for testing only?

Reply:
it is a dev-dependency in `types/Cargo.toml`

Reply:
Ok, I was looking in `Cargo.toml` in project root. Resolving.